qemu/tests/driver_in_use.py

```diff
@@ -114,13 +116,17 @@ def run_subtest(sub_type):
         stop_time = time.time() + wait_time
         while time.time() < stop_time:
             if env["bg_status"] == 1:
-                run_subtest(sub_type)
+                run_subtest(main_test)
                 break
         if stress_thread:
             stress_thread.join(timeout=timeout,
                                suppress_exception=suppress_exception)
+        if vm.is_alive():
+            run_subtest(bg_stress_test)
     elif run_bg_flag == "after_bg_test":
         run_subtest(bg_stress_test)
         if vm.is_dead():
             vm.create(params=params)
-        run_subtest(sub_type)
+        run_subtest(main_test)
+        if vm.is_alive():
+            run_subtest(bg_stress_test)
```

Comment:
What is the reason to add this if section?

Reply:
If main test is shutdown, no need to run bg_stress_test any more.

Reply:
So the main test must not be `shutdown` when `run_bg_flag == "after_bg_test"`.

I won't suggest adding this step because it is covered when `run_bg_flag == "before_bg_test"`.

Reply:
run bg_stress_test before main_test is not same as run bg_stress_test after main_test,that's why we do have these 3 scenarios.

Reply:
You're right. However, what I want to explain is we already had:

    1. first run main_test then run bg_test (the before_test case)
    2. first run bg_test then run main_test (the after_test case)

So why should we have:

    first run bg_test then run main_test and at last run bg_test

Reply:
@luckyh 
You are right.
Modified,thanks a lot.